lib/package.py

```diff
@@ -32,6 +32,7 @@
 
 # TODO: make this configurable by a base dir parameter
 BUILD_CACHE_DIR = "cache"
+REPOSITORIES_DIR_NAME = 'repositories'
```

Comment:
What repositories? I prefer PACKAGES_REPOS_TARGET_PATH , this is the name I used in PR builds#190